src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g

```diff
@@ -226,7 +226,7 @@ tokens {
 compilationUnit
     :    // A compilation unit starts with an optional package definition
         // semantic check because package definitions can be annotated
-        // which causes possible non-determinism in Antrl
+        // which causes possible non-determinism in Antlr
```

Comment:
Please remove this from [whitelist.words](https://github.com/checkstyle/checkstyle/blob/ada6faa8e2447ab6b73ae1b84e942bf98ac96053/.ci/jsoref-spellchecker/whitelist.words#L42) too.

Reply:
done